apps/data_handler/handler_tools/data_parser/serializers.py

```diff
@@ -444,4 +444,79 @@ def validate_numeric_string(cls, value: str, info: ValidationInfo) -> Decimal:
         except ValueError:
             raise ValueError(
                 f"{info.field_name} field is not a valid hexadecimal number"
-            )
\ No newline at end of file
+            )
+
+class InterestRateModelEventData(BaseModel):
+    """
+    Data model representing an interest rate model event in the Nostra protocol.
+
+    Attributes:
+        debt_token (str): The address of the debt token.
+        lending_rate (str): The lending rate in hexadecimal.
+        borrow_rate (str): The borrow rate in hexadecimal.
+        lending_index (str): The lending index in hexadecimal.
+        borrow_index (str): The borrow index in hexadecimal.
+    """
+    debt_token: str
+    lending_rate: str
```

Comment:
shouldn't be "lending_rate", "borrow_rate", "lending_index", "borrow_index" `Decimal`